core/store/models/service_agreement.go

```diff
@@ -2,21 +2,38 @@ package models
 
 import (
 	"bytes"
-	"encoding/binary"
 	"encoding/json"
-	"fmt"
 	"io"
 	"io/ioutil"
-	"math"
 	"math/big"
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/pkg/errors"
 	"github.com/smartcontractkit/chainlink/core/store/assets"
 	"github.com/smartcontractkit/chainlink/core/utils"
+	null "gopkg.in/guregu/null.v3"
 )
 
+// Encumbrance connects job specifications with on-chain encumbrances.
+type Encumbrance struct {
+	// Corresponds to requestDigest in solidity ServiceAgreement struct
+	ID uint `json:"-" gorm:"primary_key;auto_increment"`
+	// Price to request a report based on this agreement
+	Payment    *assets.Link `json:"payment" gorm:"type:varchar(255)"`
+	Expiration uint64       `json:"expiration"`
```

Comment:
Missing comment

Reply:
It's in `Coordinator.sol`. Added a comment.